docs/examples/track_fusion_example.py

```diff
@@ -0,0 +1,589 @@
+    KF_track2.update(KF_sensor_track2)
+
+# Add the tracks to the metric manager
+metric_manager.add_data({'KF_1_tracks': KF_track1,
+                         'KF_2_tracks': KF_track2,
+                         'PF_1_tracks': PF_track1,
+                         'PF_2_tracks': PF_track2}, overwrite=False)
+
+metric_manager.add_data({'KF_fused_tracks': KF_fused_track,
+                         'PF_fused_tracks': PF_fused_track,
+                         }, overwrite=False)
+
+truths = set(groundtruth_simulation.groundtruth_paths)
+metric_manager.add_data({'truths': truths}, overwrite=False)
+# %%
+# Let's visualise the various tracks and detections in the cases
+# using the Kalman and particle filters.
+plotter.plot_tracks(PF_track1, [0, 2], line=dict(color="orange"), track_label='PF partial track 1')
+plotter.plot_tracks(PF_track2, [0, 2], line=dict(color="gold"), track_label='PF partial track 2')
+plotter.plot_tracks(PF_fused_track, [0, 2], line=dict(color="red"), track_label='PF fused track')
+plotter.plot_tracks(KF_fused_track, [0, 2], line=dict(color="blue"), track_label='KF fused track')
+plotter.plot_tracks(KF_track1, [0, 2], line=dict(color="cyan"), track_label='KF partial track 1')
+plotter.plot_tracks(KF_track2, [0, 2], line=dict(color="azure"), track_label='KF partial track 2')
+
+plotter.fig
+
+
+# %%
+# 4) Evaluate the obtained tracks with the groundtruth trajectory.
+# ---------------------------------------------------------------
+# At this stage we have almost completed our example. We have created the
+# detections from the radars, performed the tracking and the
+# fusion of the tracks. Now we use the :class:`~.MetricManager`
+# to generate summary statistics on the accuracy of the tracks
+# by comparing them with the groundtruth measurements.
+#
+# If we consider the SIAP metrics, we can appreciate that
+# the fused tracks have a lower error compared to the
+# partial tracks obtained with the single instruments.
+#
+
+# Loaded the plotter for the various metrics.
+from stonesoup.plotter import MetricPlotter
+
+metrics = metric_manager.generate_metrics()
+graph = MetricPlotter()
+
+graph.plot_metrics(metrics, generator_names=['SIAP_KF_fused-truth',
+                                             'SIAP_PF_fused-truth',
+                                             'SIAP_KF1-truth',
+                                             'SIAP_KF2-truth',
+                                             'SIAP_PF1-truth',
+                                             'SIAP_PF2-truth'],
+                   color=['blue', 'red', 'cyan', 'azure', 'orange', 'gold'])
+graph.fig
+
+# %%
+# Conclusions
+# -----------
+# In this example we have shown how it is possible to
+# merge the tracks generated by independent trackers
+# ran on sets of data obtained by separate sensors. We
+# have, also, compared how the Kalman and the particle
+# filters behave in these cases, making track to truth
+# comparisons.
```

Comment:
If you add this line at the end it means the thumbnail will be the tracks plot rather than the SIAP plots which might look nicer

`# sphinx_gallery_thumbnail_number = 2`